keno/keno_input.inp

```diff
@@ -0,0 +1,431 @@
+=csas6 
+uwnr triga fuel
+ce_v7.1_endf
+Read Param 
+ flx=yes npg=100000 gen=110 nsk=10 rnd=19073486328125
+End Param 
+Read Composition  
+graphite 50002  DEN=2.23   end
+graphite 50001  DEN=2.23   end
+B-10   40001   0 0.001007273004426626  end 
+U-238 1170801  0 0.001007273004426626  end 
+U-235 1190100  0 0.0002505920710611547  end 
+U-235 1190101  0 0.0002505920710611547  end 
+U-235 1190102  0 0.0002505920710611547  end
+U-235 1190199  0 0.0002505920710611547  end 
+U-235 1190200  0 0.0002505920710611547  end 
+U-235 1190201  0 0.0002505920710611547  end 
+U-235 1190202  0 0.0002505920710611547  end
+U-235 1190299  0 0.0002505920710611547  end 
+U-235 1190300  0 0.0002505920710611547  end 
+U-235 1190301  0 0.0002505920710611547  end 
+U-235 1190302  0 0.0002505920710611547  end
+U-235 1190399  0 0.0002505920710611547  end 
+U-235 1190400  0 0.0002505920710611547  end 
+U-235 1190401  0 0.0002505920710611547  end 
+U-235 1190402  0 0.0002505920710611547  end
+U-235 1190499  0 0.0002505920710611547  end
+U-235 1190500  0 0.0002505920710611547  end 
+U-235 1190501  0 0.0002505920710611547  end 
+U-235 1190502  0 0.0002505920710611547  end
+U-235 1190599  0 0.0002505920710611547  end 
+U-235 1190600  0 0.0002505920710611547  end 
+U-235 1190601  0 0.0002505920710611547  end 
+U-235 1190602  0 0.0002505920710611547  end
+U-235 1190699  0 0.0002505920710611547  end 
+U-235 1190700  0 0.0002505920710611547  end 
+U-235 1190701  0 0.0002505920710611547  end 
+U-235 1190702  0 0.0002505920710611547  end
+U-235 1190799  0 0.0002505920710611547  end 
+Water 30000  DEN=1.0   end
+Zr-90 20000  0 0.022077113304662624  end
+Zr-90 30001  0 0.022077113304662624  end 
+Zr-91 30001  0 0.004814484184223803  end 
+Zr-92 30001  0 0.007359037768220872  end 
+Zr-94 30001  0 0.007457730403013338  end 
+Zr-96 30001  0 0.0012014755539952447  end 
+Zr-96 40002  0 0.0012014755539952447  end 
+End composition
+read geometry
+unit 3102
+com="master fuel pin"
+Zcylinder 1000 0.5 15 0 
+Zcylinder 1001 1.5 15 0
+Zcylinder 1002 2 15 0 
+plane 1100 Zpl=1 Con=-3
+plane 1101 Zpl=1 Con=-6
+plane 1102 Zpl=1 Con=-9
+plane 1103 Zpl=1 Con=-12
+plane 2000 Zpl=1 Con=0
+plane 2002 Zpl=1 Con=-1
+plane 2003 Zpl=1 Con=-14
+plane 2004 Zpl=1 Con=-15
+cuboid 1 2.5 -2.5 2.5 -2.5 15  0
+media 1190100 1 -1000 1001 1100 -1101
+media 1190101 1 -1000 1001 1101 -1102
+media 1190102 1 -1000 1001 1102 -1103
+media 1190199 1 -1001 1002 1100 -1103
+media 50002 1 1002 2003 -2004
+media 50002 1 -1001 1002 1103 -2003
+media 30001 1 1001 1103 -2003
+media 20000 1 1000 1100 -1103
+media 30001 1 1001 2002 -1100
+media 50002 1 -1001 1002 2002 -1100
+media 50001 1  1002 2000 -2002
```

Comment:
I guess this(and those in following fuel pins) was meant to be `50002` instead of `50001`.